src/XMLimport.cpp

```diff
@@ -829,7 +829,7 @@ void XMLimport::readHostPackage(Host* pHost)
         pHost->mAcceptServerMedia = (attributes().value("mAcceptServerMedia") == "yes");
     }
     pHost->mMapperUseAntiAlias = (attributes().value("mMapperUseAntiAlias") == "yes");
-    pHost->mMapperShowRoomBorders = (attributes().value("mMapperShowRoomBorders") == "yes");
+    pHost->mMapperShowRoomBorders = attributes().value("mMapperShowRoomBorders") == "yes" || !attributes().hasAttribute("mMapperShowRoomBorders");
```

Comment:
Pretty smart solution! If you're up for it, would be nice to have for other settings as well instead of the `if` wrappers

Reply:
Thanks. I can do it in separate PR, not to mess up this one. I will have more time around Christmas and without decent gaming PC, therefore with more time for coding :)